streams/src/test/java/org/apache/kafka/streams/tests/RelationalSmokeTest.java

```diff
@@ -806,20 +799,14 @@ public static boolean verifySync(final String broker, final Instant deadline) th
             }
         }
 
-        public static <T> void assertThat(final AtomicBoolean pass,
-                                          final StringBuilder failures,
-                                          final String message,
-                                          final T actual,
-                                          final Matcher<? super T> matcher) {
-            if (!matcher.matches(actual)) {
+        public static void assertThat(final AtomicBoolean pass,
+                                      final StringBuilder failures,
+                                      final String message,
+                                      final boolean passed) {
+            if (!passed) {
                 if (failures != null) {
-                    final Description description = new StringDescription(failures);
-                    description.appendText("\n" + message)
-                               .appendText("\nExpected: ")
-                               .appendDescriptionOf(matcher)
-                               .appendText("\n     but: ");
-                    matcher.describeMismatch(actual, description);
-                    description.appendText("\n");
+                    final StringBuffer description = new StringBuffer(failures);
+                    description.append("\n" + message);
```

Comment:
I was following the earlier pattern of `matcher` here, but it doesn't look like we do anything with this description.... I think it makes more sense to append this to the `failures` that we pass in. I'm not sure if `matcher` did that automatically or if this was just a gap?